src/style.css

```diff
@@ -1 +1,79 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf'), url('fonts/Avenir-Heavy.ttf');
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.search__bar {
+  margin: 20px 8px 0;
+}
```

Comment:
Be consistent with your margins. It's better to add only top or bottom margins, but not both, to avoid potential margin collapse.